documentation/typespec-end-to-end-scenarios.md

```diff
@@ -68,57 +68,67 @@ classDef grey fill:#CCCCCC,color:#555555;
 User((::))-->A
 A["clone Rest-API and SDK repo locally"] --> B
 B["2.1 pre-requsite dependencies installation
-(each language would have a installation script)"] --> C
-C["2.2 TypeSpec-Project-Sync.ps1
-(a.create tsp-location.yaml and drop to temp location
-b.update tsp-location.yaml if existed
-c.fetch specs from remote spec repo or use local spec repo)"] --> D
- D["2.3 TypeSpec-Project-Generate.ps1
+(each language would have a installation script)"] --> F
+F["2.2 TypeSpecProjectPrepare.ps1
+(a.fetch `tspconfig.yaml` from remote if it doesn’t exist locally
+b.parse `tspconfig.yaml`
+c.create sdk project folder if not existing
+d.create/update `tsp-location.yaml`)
+"] --> C
+C["2.3 TypeSpec-Project-Sync.ps1
```

Comment:
We should make these sub-boxes in the Process box so it is clear that they aren't called separately outside of that step.